config/views.yaml

```diff
@@ -67,7 +66,7 @@ component_readiness:
       enabled: true
     regression_tracking:
       enabled: true
-  - name: 4.19-ha-vs-single
+  - name: 4.19-x86-vs-arm
```

Comment:
not very exciting yet...
![image](https://github.com/user-attachments/assets/2b544aaa-5e21-430d-babf-74472cc49294)